test/testdata/lsp/fast_path/static_field_change_type__use.2.rbupdate

```diff
@@ -0,0 +1,9 @@
+# typed: strict
+# assert-fast-path: static_field_change_type__def.rb,static_field_change_type__use.rb
+extend T::Sig
+
+sig {returns(String)}
+def foo
+  STATIC_FIELD.fetch(:key, "default") + "suffix" # error: Expected `String` but found `T.any(Integer, String)`
+                                      # ^^^^^^^^ error: Expected `Integer`
```

Comment:
```suggestion
  #                                     ^^^^^^^^ error: Expected `Integer`
```

Reply:
I have a slight preference for aligning the comment with the start of the line, otherwise Vim's `=` command and code formatters tend to attempt to re-indent the comments incorrectly.